pkg/schemas/database_parameters.go

```diff
@@ -0,0 +1,47 @@
+package schemas
+
+import (
+	"slices"
+	"strings"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var (
+	ShowDatabaseParametersSchema = make(map[string]*schema.Schema)
+	databaseParameters           = []sdk.AccountParameter{
+		sdk.AccountParameterDataRetentionTimeInDays,
+		sdk.AccountParameterMaxDataExtensionTimeInDays,
+		sdk.AccountParameterExternalVolume,
+		sdk.AccountParameterCatalog,
+		sdk.AccountParameterReplaceInvalidCharacters,
+		sdk.AccountParameterDefaultDDLCollation,
+		sdk.AccountParameterStorageSerializationPolicy,
+		sdk.AccountParameterLogLevel,
+		sdk.AccountParameterTraceLevel,
+		sdk.AccountParameterSuspendTaskAfterNumFailures,
+		sdk.AccountParameterTaskAutoRetryAttempts,
+		sdk.AccountParameterUserTaskManagedInitialWarehouseSize,
+		sdk.AccountParameterUserTaskTimeoutMs,
+		sdk.AccountParameterUserTaskMinimumTriggerIntervalInSeconds,
+		sdk.AccountParameterQuotedIdentifiersIgnoreCase,
+		sdk.AccountParameterEnableConsoleOutput,
+	}
+)
+
+func init() {
+	for _, param := range databaseParameters {
+		ShowDatabaseParametersSchema[strings.ToLower(string(param))] = ParameterSchema
+	}
+}
+
+func DatabaseParametersToSchema(parameters []*sdk.Parameter) map[string]any {
```

Comment:
As mentioned [here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2872#discussion_r1652389227), let's leave it as is for now. It will be generated / changed in the pr from @sfc-gh-asawicki.